### ISSUE TITLE
Selecting card text opens link

### ISSUE THREAD

**Issue by stef-barburou** [DIS-SIN/Busrides#140]
If I select text in one of the Busrides cards (ie. if I wanted to copy it or something), it takes me to the page content even though I didn't click anything to take me there. This shouldn't happen and would not pass WCAG (I forget which failure, but it's there somewhere). To investigate.

**Comment by stef-barburou** [comment]
backlogging this until the card system is reviewed
